### ISSUE TITLE
Excluding lcobucci/jwt prevents from including other packages which relay on it

### ISSUE THREAD

**Issue by bm-srdjanmarjanovic** [auth0/laravel-auth0#234]
### SDK Version

6.5

### PHP Version

PHP 7.4

### Composer Version

2.x

### What happened?

Packages relaying on lcobucci/jwt can not be installed because it is marked as conflicting in composer.json

We are in the process of switching to the auth0 but still relay on laravel/passport which in turn requires lcobucci/jwt. 

### How can we reproduce this issue?

Require laravel/passport ^10.0 and then update or require auth0/login ^6.5  (or dev-main)  in composer.json. Composer will return an error similar to this: 

```
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires auth0/login dev-main -> satisfiable by auth0/login[dev-main].
    - laravel/passport v10.2.0 requires lcobucci/jwt ^3.4|^4.0 -> satisfiable by lcobucci/jwt[4.1.5].
    - lcobucci/jwt 4.1.5 conflicts with auth0/login dev-main.
    - laravel/passport is locked to version v10.2.0 and an update of this package was not requested.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

**Comment by rajnarayan-pix** [comment]
Getting  same issue in laravel 8 and PHP 8.

**Comment by evansims** [comment]
Hi @bm-srdjanmarjanovic 👋 Sorry for the frustration; this is a known issue at present.

The current release of our Laravel plugin uses a forked version of the mentioned library that allows us to shim in PHP 8 support. The library's author did not want to backport PHP 8 support into the version of the dependency we relied on, which required us to build a workaround in the meantime.

This dependency conflict was introduced in the upstream auth0-PHP v7, which this library relies on, to allow us to support PHP 8.0. The problematic dependency was removed in auth0-PHP v8, and we're in the process of updating this library to use that SDK version instead, which will resolve this issue. We're hoping to have a new version of this library out yet this quarter that resolves this issue.
